### DISCUSSION TITLE
Existing k3s cluster on raspberrypi

### DISCUSSION THREAD

**Discussion by techny210** [zarf-dev/zarf#1390] [General]
I have existing k3s cluster running on 4 nodes (raspberrypi 4B 8GBRAM), is there an "easy" way to build/integrate zarf into it?
I have also downloaded the arm package but there is no binaries there just bunch of tars (components, images, sboms, etc)
Where is zarf binary for raspberry, so I can zarf init and start "playing" :-) Did I miss something?
Your help is greatly appreciate it

**Comment by Racer159** [comment]
@techny210 apologies totally missed this discussion post - for arm64 you should be able to use the arm64 Linux binary from our releases: 
https://github.com/defenseunicorns/zarf/releases/tag/v0.26.0
![image](https://user-images.githubusercontent.com/3977569/233468897-02cf610f-f3ed-4a14-a9a2-01ec3475d719.png)

**Reply by techny210** [reply]
Thank You !
